src/css/custom.css

```diff
@@ -101,6 +104,8 @@ div[class^='announcementBar_'] {
   font-size: 18px;
   height: 35px;
   padding: 10px;
+  padding-top: 30px;
+  padding-bottom: 30px;
```

Comment:
By adding such change, it'll change the style in all views and this solution is not feasible at all.

![image](https://github.com/wingkwong/leetcode-the-hard-way/assets/35857179/a913a986-ad95-4ba8-b5a0-55feea249c38)

In fact, the correct changes I'm expecting are rather simple. You just need to remove all the paddings (L106 - 108) and set `height` to `auto`.

Reply:
okay,

Reply:
Done...